### ISSUE TITLE
Be able to hire villagers

### ISSUE THREAD

**Issue by Zhwible** [seymourimadeit/guardvillager-suggestions#23]
As the title says, have villagers follow you around and defend you so long as you give them emeralds.

**Comment by ChromaPIE** [comment]
Or some other ways to hire guards to follow and protect players for life long would also be good enough! Now the guards' inventory access would gone with the Hero of the Village effect, that's bad.

**Comment by skyundefined** [comment]
Yeah I like this idea too. Or more simply the guards will be passive to players if converted by them.

**Comment by Alainx277** [comment]
There's a compatible mod that does this: https://www.curseforge.com/minecraft/mc-mods/recruits
